Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 2,
+        "Minor": 172,
```

Comment:
please update the patch version, bumped up version should be 3.2.7

Reply:
I'm confused how the versioning is supposed to work. My version was certainly wrong, but I'm not sure how to correlate increasing the patch number with [the instructions](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md):

  1. Check current Azure DevOps sprint
    - Sprint is 172
  2. If sprint number differs from current minor number - set it to current sprint number, set patch to 0
    - `172 != 2` so `minor = 172` and `patch = 0`

So then it should be `3.172.0`?

Reply:
@MisinformedDNA Yes, you can use the 3.172.0. For AzureResourceManagerTemplateDeploymentV3 task, we use to increment the patch version for minor fix and minor version for some enhancements in the task and major for new features.

Reply:
Cool. Just wanted to understand. Fixed.